endpoints-support.md

```diff
@@ -86,96 +95,102 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
+|<sub>/rest/id-pools/vsn/ranges/{id}/allocated-fragments</sub>                            | GET      |  :black_circle:  | :black_circle:   | :black_circle: |
+|<sub>/rest/id-pools/vsn/ranges/{id}/allocator</sub>                                      | PUT      |  :black_circle:  | :black_circle:   | :black_circle: |
+|<sub>/rest/id-pools/vsn/ranges/{id}/collector</sub>                                      | PUT      |  :black_circle:  | :black_circle:   | :black_circle: |
+|<sub>/rest/id-pools/vsn/ranges/{id}/free-fragments</sub>                                 | GET      |  :black_circle:  | :black_circle:   | :black_circle: |
+|     **ID Pools vWWN Ranges**                                                                                                                      |
+|<sub>/rest/id-pools/vwwn/ranges</sub>                                                    | POST     |  :black_circle:  | :black_circle:   | :black_circle: |
+|<sub>/rest/id-pools/vwwn/ranges/{id}</sub>                                               | GET      |  :black_circle:  | :black_circle:   | :black_circle: |
+|<sub>/rest/id-pools/vwwn/ranges/{id}</sub>                                               | PUT      |  :black_circle:  | :black_circle:   | :black_circle: |
+|<sub>/rest/id-pools/vwwn/ranges/{id}</sub>                                               | DELETE   |  :black_circle:  | :black_circle:   | :black_circle: |
+|<sub>/rest/id-pools/vwwn/ranges/{id}/allocated-fragments</sub>                           | GET      |  :black_circle:  | :black_circle:   | :black_circle: |
+|<sub>/rest/id-pools/vwwn/ranges/{id}/allocator</sub>                                     | PUT      |  :black_circle:  | :black_circle:   | :black_circle: |
+|<sub>/rest/id-pools/vwwn/ranges/{id}/collector</sub>                                     | PUT      |  :black_circle:  | :black_circle:   | :black_circle: |
+|<sub>/rest/id-pools/vwwn/ranges/{id}/free-fragments</sub>                                | GET      |  :black_circle:  | :black_circle:   | :black_circle: |
+|     **Interconnect Link Topologies**                                                                                                             |
 |<sub>/rest/interconnect-link-topologies</sub>                                            | GET      | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/interconnect-link-topologies/{id}</sub>                                       | GET      | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   |
-|     **Interconnect Type**                                                                                                                        |
+|     **Interconnect Types**                                                                                                                        |
```

Comment:
If alphabetical order is important. Maybe it's best to put after ```Interconnects```.

Reply:
It is in alphabetical order (following sort order as in the Excel spreadsheet).